tests/bud.bats

```diff
@@ -673,7 +673,7 @@ symlink(subdir)"
   _prefetch busybox
   run_buildah 125 build -t testbud3 $WITH_POLICY_JSON $BUDFILES/dockerignore3
   expect_output --substring 'building.*"COPY test1.txt /upload/test1.txt".*no such file or directory'
-  expect_output --substring $(realpath "$BUDFILES/dockerignore3/.dockerignore")
+  expect_output --substring 'filtered out using /[^ ]*/.dockerignore'
```

Comment:
It's not obvious to me how this test change is related...did the build there try to modify the context directory?

Reply:
The name of the directory which contains the .dockerignore file and which is included in the error message is no longer the one passed on the command line, so the exact string comparison began failing.  It was either change the test to not care about the specific path, or lie about the file's location to get the test to pass as it was, so I changed the test.